src/themes/shared/lineHeights.ts

```diff
@@ -1,5 +1,5 @@
 export const lineHeights = {
-  display: '100%',
-  heading: '125%',
-  body: '150%',
+  xs: 1,
+  sm: 1.25,
+  md: 1.5,
```

Comment:
With renaming the keys like this way we are able to use them in different cases.

Reply:
The % value is a little bit tricky to read.

Reply:
this is actually also better because % can result in unexpected things https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#values